app/src/main/java/com/teamagam/gimelgimel/app/map/viewModel/IMapView.java

```diff
@@ -13,4 +13,6 @@
     void goToLocation(PointGeometry location);
 
     void addLayer(VectorLayer vectorLayer);
+
+    void openSendGeoDialog(PointGeometry pointGeometry);
```

Comment:
i am not sure take picture and gotolocatin should be here.
they are not really connected to the map